src/metabase/driver/common/parameters/values.clj

```diff
@@ -116,6 +120,12 @@
                  (sequential? value-info-or-infos) (mapv #(dissoc % :target) value-info-or-infos))
                i/no-value)})))
 
+(s/defn ^:private question-native-query-for-tag :- (s/maybe (s/cond-pre su/Map (s/eq i/no-value)))
+  "Returns the native query for the `:question` referenced by the given tag."
+  [tag :- TagParam, _params :- (s/maybe [i/ParamValue])]
+  (when-let [card-id (:question tag)]
+    (db/select-one-field :dataset_query Card :id card-id)))
```

Comment:
Why is this assumed to be a native question?

Reply:
Mostly just to get it working. I'll keep this comment unresolved to remind me that non-native questions should be converted to native, around here. Thanks! :+1: